### PR TITLE
#36 Adds detailed bug example

### PR DESCRIPTION
**Pull request by Cleop**
#36 Adds detailed bug example

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -155,6 +155,11 @@ v10 or iPhone 6 view.
 Then take a screenshot of the bug and put it all into an issue to be fixed
 :wrench: :sparkles:.
 
+**Here is an example of a well detailed bug:**
+
+![image](https://user-images.githubusercontent.com/16775804/35097023-4900501c-fc46-11e7-9856-0387d1f64d17.png)
```
Comment (nelsonic) [inline readme.md]: wish this issue/bug had a clear structure ... https://github.com/devspace/awesome-github-templates

**Review by nelsonic — COMMENTED** [review]
the _last_ thing I want as a developer is a paragraph containing all the metadata. 
This is a _better_ example: https://github.com/Polymer/polymer/issues/5047

Point is I don't think _any_ of our projects (dwyl or client) are doing "bug" reporting particularly well ..

**Comment by iteles** [comment]
@nelsonic Agreed, BUT, whilst that's a good issue as written by a _developer_ we need a good example of something that's feasible for a Product Owner to write...

**Comment by iteles** [comment]
Linked to #41

**Review by iteles — APPROVED** [review]
Merging this in based on the updated wording, we need to keep a keen eye out for more examples and do #41

**Pull request merged**